meshtastic/test.py

```
@@ -141,72 +295,102 @@ def testThread(numTests=50) -> bool:
     try:
-        iface: meshtastic.tcp_interface.TCPInterface = TCPInterface("localhost")
+        iface: TCPInterface = TCPInterface("localhost")
         iface.showInfo()
         iface.localNode.showInfo()
         iface.localNode.exitSimulator()
         iface.close()
         logger.info("Integration test successful!")
-    except:
-        print("Error while testing simulator:", sys.exc_info()[0])
-        traceback.print_exc()
+    except Exception:  # noqa: BLE001 - intentional catch-all for test exit-signaling
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)